server/user_account/serializers.py

```diff
@@ -5,11 +5,13 @@
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = CustomUser
-        fields = ['id', 'user_name', 'first_name', 'last_name', 'password', 'role', 'email', 'is_active']
+        fields = ['id', 'user_name', 'first_name', 'last_name',
+                  'password', 'role', 'email', 'is_active']
 
-    def save(self):
+    def save(self): # pylint: disable=arguments-differ
```

Comment:
Arguments differ from the overridden method. Ignored because it's just 'self'

Reply:
I don't get it, so in the method that we're overriding, what are the arguments supposed to be?

Reply:
No arguments

Reply:
I looked it up, and the original definition is the following:

`def save(self, **kwargs):`